pantheon-decoupled-auth-example.php

```diff
@@ -99,7 +117,11 @@ function pantheon_decoupled_auth_example_menu() {
 function pantheon_decoupled_auth_example_activate() {
 	pantheon_decoupled_auth_example_create_post();
 	pantheon_decoupled_auth_example_menu();
-	pantheon_decoupled_auth_example_create_application_password();
+	if (username_exists('decouple_example_user') == null) {
```

Comment:
I should have added these check on the above create post and menu too.
Let me know if these changes are needed and if I can add it here in this PR.